.github/workflows/build.yml

```diff
@@ -512,26 +512,20 @@ jobs:
       run: xvfb-run make ci
 
   build_tsan:
-    name: 'Thread sanitizer'
-    needs: check_source
-    if: needs.check_source.outputs.run_tests == 'true'
-    uses: ./.github/workflows/reusable-tsan.yml
-    with:
-      config_hash: ${{ needs.check_source.outputs.config_hash }}
-      options: ./configure --config-cache --with-thread-sanitizer --with-pydebug
-      suppressions_path: Tools/tsan/supressions.txt
```

Comment:
NOTE: there was a typo in the filename, so this patch also fixes that.

`s/supressions.txt/suppressions.txt/`

Reply:
So this is what was causing the CI job to fail: https://github.com/python/cpython/pull/123278#discussion_r1730887900